bytecode/instr.py

```diff
@@ -268,11 +269,17 @@ def lineno(self):
     def lineno(self, lineno):
         self._set(self._name, self._arg, lineno)
 
+    @property
+    def stack_effect(self):
+        # All opcodes whose arguments are not represented by integers have
+        # a stack_effect indepent of their argument.
+        arg = (self._arg if isinstance(self._arg, int) else
+               0 if self._opcode >= _opcode.HAVE_ARGUMENT else None)
```

Comment:
My point is that the computation can always be carried out even on Instr, as all instruction whose arguments is not an int does not have a stack effect depending on the argument. Furthermore as the stack depth calculation is done on a cfg we cannot force to only have ConcreteInstr. Another way would be to have a ConcreteCFG but I really do not like this idea.